app/jobs/invoices/refresh_draft_job.rb

```diff
@@ -12,7 +12,7 @@ class RefreshDraftJob < ApplicationJob
 
     unique :until_executed, on_conflict: :log, lock_ttl: 12.hours
 
-    def perform(invoice)
+    def perform(invoice:)
```

Comment:
Isn't there a risk of introducing a minor breaking change (during the deployment) with this change ? If `clock` gets deployed before `worker` and the job runs, it would serialize the argument as keyword args and the running `worker` code would still expect a global id. This could lead to some dead jobs until `worker` finish deploying.

Reply:
I would say it should be fine. Even if we end up with some dead jobs (which is low probability), this job is safe to run multiple times and we can re-run it